### ISSUE TITLE
Can a component in a sibling viewport have a child router?

### ISSUE THREAD

**Issue by brian-skinner** [angular/router#229]
**Question:**
If a path in an App's `$routeConfig` specifies sibling components (and the html file has corresponding named viewports), then is there any way for one of the sub-components to have a Child Router (as in the `CorpusController` example below)?

**In app.js:**

    angular.module('app', ['ngNewRouter'])
      .controller('AppController', ['$router', AppController]);
    
    AppController.$routeConfig = [
      { path: '/demo', components: { left: 'toolbar', right: 'corpus'} }
    ];
    function AppController ($router) {}

**In CorpusController.js:**

    angular.module('app.corpus', ['ngNewRouter'])
      .controller('CorpusController', ['$router', CorpusController]);
    
    CorpusController.$routeConfig = [
      { path: '/foo', component: 'foo' },
      { path: '/bar', component: 'bar' }
    ];
    function CorpusController ($router) {}

**Comment by btford** [comment]
Yes; this should "just work." If you're having an issue with it, can you please provide a reproduction via [jsbin](http://jsbin.com/), [plnkr.co](http://plnkr.co), or [jsfiddle](http://jsfiddle.net/) so I can take a look? Thanks!

**Comment by brian-skinner** [comment]
Thanks for the quick reply.

Here's a plunker that shows what I'm attempting to do:
http://embed.plnkr.co/v21TUIraiCiOdSUQ4eQ9/preview

I'm not sure that what I'm trying to do actually makes sense, because I don't see how the parent-level router could know which of the sibling viewports has the child-router.

Thanks for looking at this!
